packages/storybook/cxl-ui/cxl-marketing-nav.data.json

```diff
@@ -34,80 +34,217 @@
       "component": "a",
       "href": "https:\/\/conversionxl.com\/institute\/dashboard\/",
       "id": 35,
-      "parent": 0,
       "children": [
         {
           "depth": 1,
           "text": "See my roadmap",
           "component": "a",
           "href": "https:\/\/conversionxl.com\/institute\/dashboard\/roadmap\/",
-          "id": 43,
-          "parent": 35
+          "id": 351
         },
         {
           "depth": 1,
           "text": "Edit my roadmap",
           "component": "a",
           "href": "https:\/\/conversionxl.com\/institute\/roadmap\/individual\/",
-          "id": 44,
+          "id": 352,
           "parent": 35
         },
         {
           "depth": 1,
           "text": "Edit team roadmap",
           "component": "a",
           "href": "https:\/\/conversionxl.com\/institute\/roadmap\/team\/",
-          "id": 45,
+          "id": 353,
           "parent": 35
         },
         {
           "depth": 1,
           "text": "See team progress",
           "component": "a",
           "href": "https:\/\/conversionxl.com\/institute\/dashboard\/team\/",
-          "id": 46,
+          "id": 354,
           "parent": 35
         }
       ]
     },
     {
       "depth": 0,
-      "text": "Minidegrees <vaadin-icon icon=\"lumo:dropdown\"><\/vaadin-icon>",
+      "text": "Training <vaadin-icon icon=\"lumo:dropdown\"><\/vaadin-icon>",
       "component": "a",
-      "href": "https:\/\/conversionxl.com\/institute\/online-courses\/?_categories=minidegree-programs",
+      "href": "#",
       "id": 36,
       "parent": 0,
       "children": [
+        {
+          "depth": 1,
+          "text": "<b>Deep skills<b>",
```

Comment:
One more thing @freudFlintstone. PHP code strips all HTML tags for titles so we can't use `<b>` to define look of the item. Section header (Courses) here doesn't look right.

---

**Figma design**

![Screenshot 2023-08-24 at 11 41 36](https://github.com/conversionxl/aybolit/assets/5165721/95fbda8d-93aa-47c6-911c-56874d38f5c5)

---

**WPS**

![Screenshot 2023-08-24 at 11 42 58](https://github.com/conversionxl/aybolit/assets/5165721/7fc93861-a028-49a9-8cd8-eb72714e58cb)